pulp_smash/tests/rpm/cli/test_sync.py

```diff
@@ -99,3 +103,60 @@ def tearDownClass(cls):
             'pulp-admin rpm repo delete --repo-id {}'
             .format(cls.repo_id).split()
         )
+
+
+class ForceSyncTestCase(unittest2.TestCase):
+    """Test whether Pulp can force sync content by ignoring cache.
+
+    This test case targets Redmine story `Pulp #1938`_ and Pulp Smash issue
+    `Pulp Smash #301`_. The test steps are as following:
+
+    1. Create and sync a repository.
+    2. Checkout to content directory /var/lib/pulp/content/units/rpm/.
+    3. Remove some random contents in that folder.
+    4. Trigger a force sync to verify if the removed contents will be
+        restored.
+
+    .. _Pulp #1938: https://pulp.plan.io/issues/1938
+    .. _Pulp Smash #301: https://github.com/PulpQE/pulp-smash/issues/301
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        """Create and sync a repository."""
+        cls.cfg = config.get_config()
+        cls.repo_id = utils.uuid4()
+        cls.client = cli.Client(cls.cfg)
+        cls.sudo = '' if is_root(cls.cfg) else 'sudo '
```

Comment:
Pulp Smash must be able to log into the remote system either as root or as a user that can execute `sudo ...` without a password. It's the user's job to make that happen.